DuckDuckGo/ViewVisibility.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  ViewVisibility.swift
+//  DuckDuckGo
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftUI
+
```

Comment:
I got this from the macOS codebase. Ideally all these generic extensions should be in BSK. When are we switching to the monorepo? :D I can move it to BSK now or wait. wdyt?

Reply:
Let's wait until right at the end. I think doing BSK stuff at this point will just make things more complicated than we need.